src/codeu/chat/client/ClientUser.java

```diff
@@ -14,10 +14,12 @@
 
 package codeu.chat.client;
 
-import java.util.Arrays;
 import java.util.Collection;
 import java.util.HashMap;
 import java.util.Map;
+import java.util.Arrays;
+import java.security.*;
```

Comment:
try not to import * unless you need more than 100 files. This is for code longevity - if oracle decides to add a class called User in java.security, then all of a sudden your code will stop compiling...